### ISSUE TITLE
Cracks in the voice

### ISSUE THREAD

**Issue by sambalshikhar** [gpt-omni/mini-omni2#54]
Hi , i noticed that there are minor some minor cracks and glitches in between when the audio streaming , is that a Pyaudio issue ?

**Comment by mini-omni** [comment]
not really, I suppose it's more likely caused by SNAC, as SNAC does not support streaming decoding. One may fix it by retraining the model with a streaming audio codec method like mimi from moshi.
